test/import.js

```diff
@@ -44,17 +46,18 @@ exports.import = {
         test.done();
     },
 
-    "supports a Date instance": function(test) {
+    "supports a Date instance greater than 5 days old": function(test) {
```

Comment:
It would be useful to have a test verifying that the `import()` method remains working even with dates more recent than 5 days ago (for instance, to guard against a future extraction of the time-threshold logic into `send_request()`).